package/base-files/files/bin/board_detect

```diff
@@ -1,14 +1,20 @@
 #!/bin/sh
 
-CFG=$1
+CFG="${1:-/etc/board.json}"
```

Comment:
Quoting is redundant here iirc. Assignments in the form `b=$a` should be safe, even if `$a` contains white spaces etc.

Reply:
While technically correct, better be consistent and 'always quote your variables'. Shellcheck will even tell you 'quote to avoid word-splitting', So instead of relying on specific rules (should work in this case because of a, b or c) just always quote it, and avoid any potential issues or bugs. Especially those like 'oh but I thought this ...' Just keep things consistent and predictable.